ci-operator/step-registry/ipi/conf/aws/oidc-creds-provision/ipi-conf-aws-oidc-creds-provision-commands.sh

```diff
@@ -64,3 +68,12 @@ echo -e "\n"
 # copy generated secret manifests from ccoctl target directory into shared directory
 cd "/tmp/manifests"
 for FILE in *; do cp $FILE "${MPREFIX}_$FILE"; done
+
+# for Shared VPC install, same all role names, they will be used in trust policy
+for f in openshift*.yaml
+do
+  [[ -e "$f" ]] || exit 1
+  role_arn=$(yq-go r ${f} 'stringData.credentials' | grep role_arn | awk '{print $3}')
```

Comment:
looks like the use of `yq-go` here is causing the failure in the sts rehersal: 
https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/39946/rehearse-39946-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-aws-sts/1668629045413154816/build-log.txt

Reply:
fixed, removed `yq-go`